app/views/pets_applications/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<section class="applicant_information">
+<%= render partial: '/layouts/applicant', locals: {
+  name: @applicant.name,
+  street_address: @applicant.street_address,
+  city: @applicant.city,
+  state: @applicant.state,
+  zip_code: @applicant.zip_code,
+  description: @applicant.description
+} %>
+</section>
+
+<section class="search_adopt">
+<% if @application.status == "In Progress" %>
+<h2>Add a Pet to this Application</h2>
+<%= form_with url: "/pets_applications/#{@application.id}", method: :get, data: { turbo: false } do |form| %>
```

Comment:
I am wondering if this route could be more RESTful if we did
```
"/applications/:id/pets"
```
That way we are associating an application with a pet by passing in both ids